RecoBTag/SoftLepton/plugins/SoftPFElectronTagInfoProducer.cc

```diff
@@ -109,7 +113,7 @@ void SoftPFElectronTagInfoProducer::produce(edm::Event& iEvent, const edm::Event
   			properties.ratioRel = recoelectron->p4().Dot(jetRef->p4()) / pjet.Mag2();
   			properties.p0Par    = boostedPPar(recoelectron->momentum(), jetRef->momentum());
 			properties.elec_mva    = recoelectron->mva_e_pi();
-			 if(std::abs(properties.sip3d>MaxSip3D)) continue;
+			 if(std::abs(properties.sip3dsig>MaxSip3Dsig)) continue;
```

Comment:
Good catch! I guess the original intention was to have `std::abs(properties.sip3dsig)>MaxSip3Dsig`. In practice I suspect the impact of this bug is negligible but still it should be fixed. Thanks again for catching it.